src/main/java/edu/ie3/util/quantities/QuantityUtil.java

```diff
@@ -31,6 +34,25 @@ private QuantityUtil() {
     return Quantities.getQuantity(quantity.getValue(), quantity.getUnit());
   }
 
+  /**
+   * Compares two {@link Quantity}s, if they are considerably equal. This is foremost important for
+   * {@link tech.units.indriya.quantity.NumberQuantity}s with internal big decimal or double values.
+   * The comparison is made on the absolute difference of both quantities' value. Both quantities
+   * are converted into a's unit before the comparison. Internally calls {@link
+   * QuantityUtil#isAbsolutelyEquivalent(Quantity, Quantity, double)} with a default tolerance of
+   * {@value DEFAULT_TOLERANCE}
+   *
+   * @param a First quantity to compare
+   * @param b Second quantity to compare
+   * @param <Q> Type of Quantity
+   * @return true, if both quantities' values differ less then the given tolerance else false (based
+   *     on a's unit)
+   */
+  public static <Q extends Quantity<Q>> boolean isAbsolutelyEquivalent(
```

Comment:
```suggestion
  public static <Q extends Quantity<Q>> boolean isEquivalentAbs(
```

Sorry to once again popping in with some kind of esoterically discussions about naming. As the attribute "absolutely" is also used more or less colloquial (e.g. "Your two twins look absolutely the same."), the method name could be a bit misleading. How do you feel with my suggestion?